packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -106,6 +107,11 @@ export interface IDropdownProps extends React.Props<Dropdown> {
   required?: boolean;
 
   /**
+   * Custom properties for Dropdown's Callout.
+   */
+  calloutProperties?: ICalloutProps;

```

Comment:
I think this would make sense, but it should be called `calloutProps` which is consistent with `iconProps` used elsewhere.

I think there are other cases that are interesting that this solves; for example, you want a data-automation-id on the callout.